conda/core/index.py

```diff
@@ -174,7 +174,7 @@ def _supplement_index_with_system(index):
     """
     registered_names = []
     pm = get_plugin_manager()
-    for package in chain(*pm.hook.conda_virtual_packages()):
+    for package in pm.get_registered_plugins("virtual_packages"):
```

Comment:
I know this is not part of  this PR but a thought: should all these "magic strings" be part of a predefined Enum?

Reply:
Yeah, probably, the name is derived from the hookspec we specify, not sure how to best maintain it, or if this is needed for the first launch though